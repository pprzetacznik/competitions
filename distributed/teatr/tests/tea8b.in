100000000 2 100 -1 1000000 6413902146052353875
