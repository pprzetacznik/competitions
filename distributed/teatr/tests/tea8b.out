4999995000000000
